Sources/Kitura/Router.swift

```diff
@@ -18,7 +18,12 @@ import KituraNet
 import KituraSys
 import LoggerAPI
 import Foundation
+
+#if Xcode
+import Kitura_TemplateEngine
+#else
 import KituraTemplateEngine
+#endif
```

Comment:
This doesn't seem correct, once https://github.com/apple/swift-package-manager/pull/337 is merged, the module name will be `Kitura_TemplateEngine` for both SwiftPM and Xcode.

Some context: 
Xcode converts all invalid characters in a module name to `_` where as SwiftPM was removing them instead. That PR will match the module name with Xcode's.